src/candela/components/Geo/index.js

```diff
@@ -0,0 +1,39 @@
+import geojs from 'geojs/geo.js';
+import VisComponent from '../../VisComponent';
+
+export default class Geo extends VisComponent {
+  constructor (el, options) {
+    super(el);
+
+    const geojs_opt = options.geojs || {};
+
+    this.plot = geojs.map(Object.assign({
+      node: el
+    }, geojs_opt));
+
+    this.plot.createLayer('osm', {
+      renderer: null
```

Comment:
@manthey mentioned that this may not be the best choice: I think we can tell GeoJS that it's a point layer, and it will figure out the best renderer for us.